conf/boxSdkConfig.json

```diff
@@ -0,0 +1,12 @@
+{
```

Comment:
Not sure if this is needed anymore - need to revisit

Reply:
added originally in an attempt to create a client with authority to modify the sharability of a file in order to produce a link that could be used for deep linking (in the context provided by EA to slack, give a link to link back to file). Seems at this point Box doesn't support this for skills apps - but might be able to create a different app type that still functions as a skill (see original poc before box added apps of type skill).